subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -9042,6 +9050,15 @@ void radio_rx_dequeue(void)
 		LL_ASSERT(0);
 		break;
 	}
+
+	if (radio_pdu_node_rx->hdr.type == NODE_RX_TYPE_CONNECTION) {
+		u8_t bm = ((u8_t)_radio.scanner.is_enabled << 1) |
+			  _radio.advertiser.is_enabled;
+
+		if (!bm) {
+			ll_adv_scan_state_cb(0);
```

Comment:
can this create spurious callbacks? i.e. calling ll_adv_scan_state_cb() when there has been no bitmap transition. I would like to avoid that if possible to remove state from the fitlering